src/Artsy/Relay/createEnvironment.ts

```diff
@@ -10,11 +10,23 @@ import { NetworkError } from "Utils/errors"
 
 import {
   cacheMiddleware,
+  errorMiddleware,
   loggerMiddleware,
   RelayNetworkLayer,
   urlMiddleware,
 } from "react-relay-network-modern"
 
+const isServer = typeof window === "undefined"
+const isDevelopment =
+  (isServer ? process.env.NODE_ENV : sd.NODE_ENV) === "development"
+
+// Only log on the client during development
+const loggingEnabled = isDevelopment && !isServer
```

Comment:
Why not on the server in development?

Reply:
Agree, aren't there cases where logging would be of interest on the server side?

Reply:
It's pretty noisy, especially when working in force. Since most of these logs are duplicated when we work in Storybooks on the client figured it was unnecessary and didn't feel too useful. I think a PR back to react-relay-network-modern that trims the amount of data being sent down the wire might help in this regard.